packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1273,11 +1276,15 @@ function commitSuspenseComponent(finishedWork: Fiber) {
   } else {
     newDidTimeout = true;
     primaryChildParent = finishedWork.child;
-    if (newState.timedOutAt === NoWork) {
+    if (newState.fallbackUpdate === NoWork) {
       // If the children had not already timed out, record the time.
       // This is used to compute the elapsed time during subsequent
       // attempts to render the children.
-      newState.timedOutAt = requestCurrentTime();
+      // We model this as a normal pri expiration time since that's
+      // how we infer start time for updates.
+      newState.fallbackUpdate = computeAsyncExpirationNoBucket(
```

Comment:
Naming nit: `fallbackTime`?

Reply:
I went with `fallbackExpirationTime`. I can't think of anything good. It's as if I made an update to switch to the content state at this time.